crates/rnote-engine/src/store/selection_comp.rs

```diff
@@ -126,8 +130,14 @@ impl StrokeStore {
             .collect::<Vec<StrokeKey>>();
 
         // Offsetting the new selected stroke to make the duplication apparent
-        self.translate_strokes(&new_selected, Stroke::IMPORT_OFFSET_DEFAULT);
-        self.translate_strokes_images(&new_selected, Stroke::IMPORT_OFFSET_DEFAULT);
+        // check if snap position is activated or not here
+        let offset = if snap_mode {
+            pattern_size
```

Comment:
It often is the case that the selection is not snapped to the grid even when `snap_positions` is active.
It would be better if the logic does not simply use the grid's size as offset but actively snap the duplicated selection
to the grid when it is active (maybe "default offset" + nearest grid edge?).